racer.el

```diff
@@ -231,7 +231,7 @@ foo(bar, |baz); -> foo|(bar, baz);"
   :lighter " racer"
   :keymap racer-mode-map
   (setq-local eldoc-documentation-function #'racer-eldoc)
-  (make-local-variable 'completion-at-point-functions)
+  (set (make-local-variable 'completion-at-point-functions) nil)
```

Comment:
Why not use `setq-local` as in `(setq-local completion-at-point-functions nil)`? Like on the line above it.